tests/MongoDB.Bson.TestHelpers/JsonDrivenTests/AspectAsserter.cs

```diff
@@ -21,6 +21,11 @@ public abstract class AspectAsserter
     {
         // public methods
         public abstract void AssertAspects(object actualValue, BsonDocument aspects);
+
+        public virtual void ConfigureTemplateNames(string[] templates)
```

Comment:
This name makes little sense to me.

What's a template?

Reply:
The template value in json tests:
https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/48/files#r309889101

Reply:
I've searched the various specs that use `42` in the JSON files and it seems like these values are most commonly called "place holders".